subsys/bluetooth/scan.c

```diff
@@ -18,6 +18,16 @@ LOG_MODULE_REGISTER(nrf_bt_scan, CONFIG_BT_SCAN_LOG_LEVEL);
 	BT_SCAN_SHORT_NAME_FILTER | BT_SCAN_APPEARANCE_FILTER | \
 	BT_SCAN_UUID_FILTER | BT_SCAN_MANUFACTURER_DATA_FILTER)
 
+#ifndef CONFIG_BT_SCAN_CONN_ATTEMPTS_FILTER_LEN
+#define CONFIG_BT_SCAN_CONN_ATTEMPTS_FILTER_LEN 0
+#endif
+#ifndef CONFIG_BT_SCAN_BLOCKLIST_LEN
+#define CONFIG_BT_SCAN_BLOCKLIST_LEN 0
+#endif
+#ifndef CONFIG_BT_SCAN_CONN_ATTEMPTS_CNT
+#define CONFIG_BT_SCAN_CONN_ATTEMPTS_CNT 0
+#endif
+
 /* Scan filter add mutex. */
 K_MUTEX_DEFINE(scan_add_mutex);
```

Comment:
Update the comment, we can also rename it to `scan_mutex`.

Reply:
Done